script/buildSQLGeom.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+"""
+This module will accept an input text file with (x,y,z) coordinates in a CSV-format list
+and generate a SQL string that will add these coordinates as a PostGIS Polygon
+to sites.geometry within a BETYdb instance.
+
+Command line usage:
+        -h      show help
+        -i      input filename
+        -e      EPSG reference number, default is 3857
```

Comment:
shouldn't default be 4356 / WGS84 (#96). I don't know if this is a good idea, but I believe it is the default in BETYdb.